product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx

```diff
@@ -48,10 +48,10 @@ Alternatively, you can run the script in [Azure Cloud Shell](https://azure.micro
 The syntax of the command is:
 
 ```
-curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/create-spn.sh | bash -s \
-  -- -display-name <display-name> \
-  -- -subscription <subscription> \
-  -- -years <years>
+curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/create-spn.sh | bash -s -- \
+--display-name "hello-s" \
```

Comment:
I think we are mixing syntax and examples by using actual values for the parameters in this section. I think we should use the <user_supplied_info> convention in the syntax section. There is an actual example farther down in the page where we can use specific values.